src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareService.cs

```diff
@@ -73,9 +83,14 @@ await requestContext.SendResult(new SchemaCompareResult()
                             Differences = operation.Differences
```

Comment:
Will operation.diff be available here in all cases (considering async/failure without exception if any)?

Reply:
It should be ok if operation.Differences is null as long as it's handled properly in the ADS extension